doc/api/next_api_changes/deprecations/28074-TS.rst

```diff
@@ -0,0 +1,9 @@
+``boxplot`` and ``bxp`` *vert* parameter, and ``boxplot.vertical`` rcparam
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The parameter *vert: bool* has been deprecated on `~.Axes.boxplot` and
+`~.Axes.bxp`. It is replaced by *orientation: {"vertical", "horizontal"}*
+for API consistency.
+
+The *boxplot.vertical* rcparam which controlled the orientation of ``boxplot``
+is deprecated as the new API does not require its use.
```

Comment:
```suggestion
``rcParams["boxplot.vertical"]``, which controlled the orientation of ``boxplot``,
is deprecated without replacement.
```
It's an idependent design decision that it's not reasonable/necessary to control the orientation via rcParams.